R/metadata-extraction_imagery_perimage.R

```diff
@@ -118,11 +118,15 @@ extract_rtk_fix = function(exif) {
 #' @export
 
 extract_accuracy = function(exif) {
-  exif = st_drop_geometry(exif)
-  if ("GPSXYAccuracy" %in% colnames(exif)) {
+  exif = sf::st_drop_geometry(exif)
+  # This field may not be actually present in the data but only introduced by
+  # row_binding across multiple platforms
+  if ("GPSXYAccuracy" %in% colnames(exif) && !all(is.na(exif$GPSXYAccuracy))) {
     accuracy_x = exif$GPSXYAccuracy
     accuracy_y = exif$GPSXYAccuracy
-  } else {
+  } else{
+    # Try extracting the deviation from the RTK field
+    # TODO should we instead prioritize the RTK value if both are present?
```

Comment:
Is there a situation where both the `GPSXYAccuracy` and `RtkStdLon/RtkStdLat` fields would be set?

Reply:
I tried but I could not find any EXIF that had `GPSXYAccuracy` at all! I'm not sure where this came from. For non-RTK, the best potential option I see is `circleofconfusion` and it can exist along with RtkStdLon and be different. I'm not sure what to do about this but I think for now it's OK to focus on reporting the RTK accuracy if RTK and no accuracy otherwise. Accuracy for instantaneous point coordinates is pretty tenuous anyway I think.

Reply:
It looks like the eBee reports `GPSXYAccuracy` but I didn't see it for the DJI platforms. However, all of them report the RTK metric for the NRS data I was looking at.

I also saw `circleofconfusion` but I don't think it's what we want. It appears to be an optics term related to the magnitude of the imaging blur: [wiki](https://en.wikipedia.org/wiki/Circle_of_confusion). This is supported by the fact that the values are sub-millimeter.